website/content/en/docs/concepts/sites.md

```diff
@@ -57,55 +60,102 @@ You will need the CA certificate (but **not** the CA private key) and the site c
-    and the CA and site certificate and key are set as Kubernetes secrets
-    in the namespace. You can confirm the successful completion of the step using
-    the following commands (TODO: describe output):
+After the operator is installed, you can deploy ClusterLink by applying
+ the ClusterLink instance CRD. This will cause the ClusterLink operator to
+ attempt reconciliation of the actual and intended ClusterLink deployment.
+ By default, the operator will install the ClusterLink control and data plane
+ components into a dedicated and privileged namespace (defaults to `clusterlink-system`).
+ Site wide configurations, such as the list of known `Peers`, are also maintained
+ in the same namespace.
 
-    ```sh
-    kubectl get crds
-    ```
+Refer to the [getting started guide]({{< ref "getting-started#setup" >}}) for a description
+ of the ClusterLink instance CRD's fields.
 
-    and
+<!-- TODO expand the sample CRD file? -->
 
-    ```sh
-    kubectl get secret --namespace clusterlink-operator
-    ```
+## Add or remove Sites
 
-1. Deploy ClusterLink CRD instance.
+{{< notice info >}}
+This operation is typically done by a local **Site administrator**, usually different
+ than the **Fabric administrator**.
+{{< /notice >}}
 
-    After the operator is installed, you can deploy ClusterLink by applying
-    the ClusterLink instance CRD.
-    Refer to the [getting started guide]({{< ref "getting-started#setup" >}}) for a description
-    of the CRD instance fields.
+Adding and removing sites is done by creating and deleting `Peer` CRD instances
+ in the dedicated ClusterLink namespace (typically, `clusterlink-system`). Peers are
+ added to the ClusterLink namespace by the site administrator. Information
+ regarding peer gateways and attributes is communicated out of band (e.g., provided
+ by Site or Fabric administrators over email). In the future, these may be configured
+ via a management plane.
+
+There are two fundamental attributes in the Peer CRD: the Peer's name and the list of
+ ClusterLink gateway endpoints through which the remote site's Services are available.
+ Peer names are unique and must align with the Subject name present in their certificate
+ during connection establishment. The name is used by importers in referencing an export
+ (see [here]({{< ref "services" >}}) for details).
+
+Gateway endpoint would typically be a implemented via a `NodePort` or `LoadBalancer`
+ Kubernetes Service. A `NodePort` Service would typically be used in local deployments
+ (e.g., when running in KIND clusters during development) and a `LoadBalancer` Service
+ would be used in Cloud based deployments. These can be automatically configured and
+ created via the [operator CRD]{{< ref "#deploy-clusterlink-via-the-operator-and-clusterlink-crd" >}}.
+ Not having any gateways is an error and will be reported in the `Peer`'s Status.
+ In addition, the Status could display additional Peer conditions, such a `Reachable`
```

Comment:
why could? it is going to display this..

Reply:
the conditions are not decided yet so I did not want to commit ;-)